packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -103,6 +104,37 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits }) =>
     setFilter(selectedItem);
   };
 
+  const handleDeleteEncounter = React.useCallback(
+    (encounterUuid: string, encounterTypeName?: string) => {
+      const close = showModal('delete-encounter-modal', {
+        close: () => close(),
+        encounterTypeName: encounterTypeName || '',
+        onConfirmation: () => {
+          const abortController = new AbortController();
+          deleteEncounter(encounterUuid, abortController)
+            .then(() => {
+              showToast({
+                title: t('deleteEncounter', 'Delete Encounter', { encounter: encounterTypeName }),
```

Comment:
Same here.

Also, I guess the expected title should be '{Encounter} encounter deleted' instead of 'Delete Encounter'